app/src/main/java/org/cru/godtools/activity/BasePlatformActivity.kt

```diff
@@ -318,4 +321,14 @@ abstract class BasePlatformActivity : BaseDesignActivity(), NavigationView.OnNav
 
     private fun launchTrainingTutorial() = startTutorialActivity(PageSet.TRAINING)
     // endregion Navigation Menu actions
+
+    open fun trackOpenedTool() {
+        val eventBus = EventBus.getDefault()
+        if (settings.isFeatureDiscovered(Settings.FEATURE_FIRST_TOOL_OPENED)) {
+            eventBus.post(ToolOpened)
+        } else {
+            settings.setFeatureDiscovered(Settings.FEATURE_FIRST_TOOL_OPENED)
+            eventBus.post(FirstToolOpened)
+        }
+    }
```

Comment:
this is the wrong spot to record these events, it completely misses opening tools from deep links (including shortcuts)